valhalla/baldr/openlr.h

```diff
@@ -138,8 +138,8 @@ struct LocationReferencePoint {
         latitude(!prev ? integer2decimal(decimal2integer(latitude))
                        : prev->latitude +
                              (std::round((latitude - prev->latitude) * geom_scale) / geom_scale)),
-        bearing(integer2bearing(bearing2integer(bearing))), frc(frc), fow(fow),
-        distance(integer2distance(distance2integer(distance))), lfrcnp(lfrcnp) {
+        bearing(integer2bearing(bearing2integer(bearing))),
+        distance(integer2distance(distance2integer(distance))), frc(frc), lfrcnp(lfrcnp), fow(fow) {
```

Comment:
changing the order of property construction to match order in struct (was a warning from compiler).